#Camera topic here
camera_topic: "/videofile/image_raw"
queue_size: 10

# Set your detection topic here
detection_topic: "/darknet_ros/bounding_boxes"

# Set your output topic here
tracker_topic: "/sort_track"

# Threshold for Costs between detections and tracks
cost_threhold: 15

# threshold for bounding box overlap
iou_threshold: 0.3

# image display
display: True

# min_hits
min_hits: 3

# max age
max_age: 1

image_view:

  enable_opencv: true
  wait_key_delay: 1
  enable_console_output: true
